docs/developer/app-store/apps/adyen/overview.mdx

```diff
@@ -266,3 +266,34 @@ mutation TransactionProcess($transactionId: ID!, $data: JSON, $customerIpAddress
   }
 }
 ```
+
+## Data processing
+
+The app *stores* and *processes* data that may be PIIs or sensitive:
```

Comment:
Would be good to maybe not combine these two together. We Process (or rather send) PII, but we do not store it. In terms of privacy this is a big deal, so would be cool to make the differentiation in this sentence, just not to scare anyone.

Reply:
Updated in [`87d749e` (#1415)](https://github.com/saleor/saleor-docs/pull/1415/commits/87d749ec4b8c74491d881efe4c1bbe1d7fc601e5)